### ISSUE TITLE
Feature: Render LaTeX in more pages

### ISSUE THREAD

**Issue by aprzn123** [aprzn123/TheThirdCan#8]
Currently, we inject MathJax into forum pages to render LaTeX there, but not any other pages. To be decided: which other pages should get it? Current thoughts are ask/answer/message, but not homepage.

**Comment by JackDyre** [comment]
i agree with that, what about bios? do you think bios should be able to render latex?

**Comment by thekifake** [comment]
Does LaTeX render in quotes as well? Haven't seen it yet.

**Comment by HumanoidSandvichDispenser** [comment]
Just to make sure, I think whoever will implement this should make a separate JS script and manifest.json content_script entry for injecting MathJax.
